runner/test/Quickstrom/PureScriptTest.purs

```diff
@@ -101,3 +101,11 @@ tla4 = true && always true
 tla5 = always (next true)
 
 tla6 = always (next false || next true)
+
+tla7 = until false true
+
+tla8 = until false false
+
+tla9 = until true false
+
+tla10 = until true (testOneQuery == "baz")
```

Comment:
Maybe you can add one example like `always (until p q)` as discussed above?

Reply:
Do you still feel this is necessary?

Reply:
Nope. :slightly_smiling_face: